tests/dedukti.ml

```diff
@@ -203,3 +204,33 @@ module Meta = struct
       (import_arguments @ log_dkmeta @ arguments)
       ~preprocess ~postprocess
 end
+
+module Pretty = struct
+  type argument = Import of string
```

Comment:
What is this argument for?

Reply:
To typecheck the resulting file (it's documented in the mli)